main.cpp

```diff
@@ -272,19 +326,29 @@ struct Amp
     float power = 200.f;
     float impedence = 200.f;
     int outputs = 4;
+    Amp();
     void projectAudio();
     void powerSpeakers();
     void protectSystemFailure();
 };
 
+Amp::Amp()
+{
+    brand = "";
+    modelNum = 0;
+    power = 0;
+    impedence = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise